src/profile/utils.js

```diff
@@ -44,15 +44,29 @@ function getDataItem(data, itemName) {
   }
 }
 
-function displayList(data, userInfoList) {
-  for (const listItem in data) {
-    const innerText = `${formatPropertyField(listItem)}: ${getDataItem(
-      data,
-      listItem,
-    )}`;
+function displayList(primaryData, userInfoList, secondaryData) {
+  const prevSibling = userInfoList.previousSibling.innerHTML;
+  const isOldData = prevSibling.toLowerCase().includes('old');
+  const isNewData = prevSibling.toLowerCase().includes('new');
+
+  for (const listItem in primaryData) {
+    const oldValue = getDataItem(primaryData, listItem);
+    const newValue = getDataItem(secondaryData, listItem);
+    const isValueEqual = String(oldValue).trim() === String(newValue).trim();
+
+    let diffClass;
+    if (!isValueEqual) {
+      diffClass = isOldData ? 'oldDiff' : 'newDiff';
+    }
+
+    const innerHTML = `
+      <span>${formatPropertyField(listItem)}:</span>
+      <span class=${diffClass}>${getDataItem(primaryData, listItem)}</span>
```

Comment:
Can you try using js create element method

Reply:
Refactored the same with createElement syntax. Please check it.